src/views/homepage/components/page-title/page-title.module.css

```diff
@@ -0,0 +1,17 @@
+.root {
+	color: var(--token-color-foreground-high-contrast);
+	font-size: 4.375rem;
+	font-weight: var(--token-typography-font-weight-bold);
+	line-height: 1.2;
```

Comment:
Is this a unique new typestyle? Should we codify it higher up as a global class?

Reply:
It is! We don't have any tokens or classes that go this far up in `font-size`, `line-height`, etc. I think since this is just for one page, we don't need to lift it to a global scope.

Reply:
Maybe we could name it `h0` or something and capture the styles in a class with a note that this is an exception to the HDS type styles.

Reply:
I'm not convinced at the moment that this warrants a global class helper, but I can add a comment in this code with some context!